### PR TITLE
Doc: Prepare for 1.11.0-RC1 release

### PR DESCRIPTION
**Pull request by TrueBrain**
## Motivation / Problem

It is almost time for our first (and possibly only) release candidate of the 1.11.0 series.

This would introduce the release branch and invoke a feature-freeze on 1.11. So we better be sure to have everything in there before we merge this :)

## Description

Went through the git log, deduplicated, formatted, the usual.

## Limitations

- Currently there is one open feature we would like to include in 1.11 (zstd / #8845) but hasn't seen any reviews yet. I propose if it doesn't land the 14th, we go ahead with RC1 and the feature freeze. This means zstd would be bumped to 1.12.

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,38 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Add: Make mNewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
```
Comment (James103) [inline changelog.txt]: Why do you have `mNewGRF Scanner` instead of just `NewGRF Scanner`?

Comment (TrueBrain) [reply changelog.txt]: Because people make typos? No need to be mean about it :P

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)
+Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)
+Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
+Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
+Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)
+Fix: Scale PIP-padding the same as regular padding (#8829)
+Fix #8817: Keep NewGRF order for object class sorting (#8818)
+Fix #8784: Using alt+enter didn't update the fullscreen toggle visibly (#8820)
+Fix #8808: [OSX, OpenGL] Crash on switching blitters due to double-mapping the video buffer (#8822)
+Fix: Broken company colours for 40bpp-blitter (#8821)
```
Comment (michicc) [inline changelog.txt]: This is actually a fix for `Fix #8774: Black screenshots when using 40bpp-blitter (#8791)`, so doesn't really need to be in  the changelog.

**Review by michicc — DISMISSED** [review]

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,38 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Add: Make NewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
+Add: Display refresh rate game option (#8813)
```
Comment (LordAro) [inline changelog.txt]: Should be Feature

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+Add: [Video] Move GameLoop into its own thread (#8741)
```
Comment (LordAro) [inline changelog.txt]: Is a codechange with (almost) no visible user impact, remove

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
+Add: Option to (dis-)allow accelerated video drivers (#8819)
```
Comment (LordAro) [inline changelog.txt]: Feature?

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,38 @@
...
+Add: Filter for "engines only" in build train window (#8733)
+Add: Malaysia Ringgit as Currency (#8783)
```
Comment (LordAro) [inline changelog.txt:9]: Wasn't this in beta2?

File: changelog.txt [hunk 7]
```diff
@@ -1,3 +1,38 @@
...
+Change: Don't show global goals in company goal windows (#8709)
+Change: Clarify what effect town interactions have (#8744)
+Change: De-limit framerate window's framerate (#8772)
+Fix #8855: If bootstrap failed, it could end with an empty screen instead of error (#8856)
+Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)
+Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)
+Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
+Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
+Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)
+Fix: Scale PIP-padding the same as regular padding (#8829)
```
Comment (LordAro) [inline changelog.txt]: PIP is internal, is there some more user-friendly term for it?

File: changelog.txt [hunk 8]
```diff
@@ -1,3 +1,38 @@
...
+Fix #8817: Keep NewGRF order for object class sorting (#8818)
+Fix #8784: Using alt+enter didn't update the fullscreen toggle visibly (#8820)
```
Comment (LordAro) [inline changelog.txt]: Alt+Enter

**Review by LordAro — CHANGES_REQUESTED** [review]
List is not sorted "according to spec" - feature/add/change/fix, then reverse PR order

File: changelog.txt (hunk 6 above)
Comment (TrueBrain) [reply changelog.txt:9]: Indonesia Rupiah was ;)

File: changelog.txt [hunk 9]
```diff
@@ -1,3 +1,37 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Option to (dis-)allow accelerated video drivers (#8819)
```
Comment (LordAro) [inline changelog.txt]: "GPU accelerated" ?

File: changelog.txt [hunk 10]
```diff
@@ -1,3 +1,37 @@
...
+Feature: Display refresh rate game option (#8813)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
```
Comment (LordAro) [inline changelog.txt]: Tricky thing to summarize to a single line, but still feels a bit off to me
Something like "Add setting to define maximum sprite zoom level", maybe?

File: changelog.txt [hunk 11]
```diff
@@ -1,3 +1,37 @@
...
+Add: Make NewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
```
Comment (LordAro) [inline changelog.txt]: "more smooth" -> "smoother"
"them" -> "it"

File: changelog.txt [hunk 12]
```diff
@@ -1,3 +1,37 @@
...
+Add: Malaysia Ringgit as Currency (#8783)
+Add: Filter for "engines only" in build train window (#8733)
```
Comment (LordAro) [inline changelog.txt]:
```suggestion
Add: "Engines only" filter in build train window (#8733)
```

File: changelog.txt [hunk 13]
```diff
@@ -1,3 +1,37 @@
...
+Change: De-limit framerate window's framerate (#8772)
+Change: Clarify what effect town interactions have (#8744)
```
Comment (LordAro) [inline changelog.txt:12]: Minor enough to remove?

File: changelog.txt [hunk 14]
```diff
@@ -1,3 +1,37 @@
...
+Change: Don't show global goals in company goal windows (#8709)
+Change: Recolour graph windows to brown (#8700)
+Fix #8855: If bootstrap failed, it could end with an empty screen instead of error (#8856)
```
Comment (LordAro) [inline changelog.txt]: Reverse it a bit - 
```suggestion
Fix #8855: Bootstrap could result in an empty screen on error (#8856)
```

File: changelog.txt [hunk 15]
```diff
@@ -1,3 +1,37 @@
...
+Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)
+Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)
```
Comment (LordAro) [inline changelog.txt]: "Incorrect drawing order of tram catenary sprites"

File: changelog.txt [hunk 16]
```diff
@@ -1,3 +1,37 @@
...
+Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
+Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)
+Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
+Fix: Scale padding between elements the same as regular padding (#8829)
+Fix #8808: [OSX, OpenGL] Crash on switching blitters due to double-mapping the video buffer (#8822)
+Fix #8784: Using Alt+Enter didn't update the fullscreen toggle visibly (#8820)
+Fix #8817: Keep NewGRF order for object class sorting (#8818)
+Fix #8809: Crash when removing airport when hangar window open (#8815)
+Fix #8799: Crash when Search Internet in Multiplayer (#8801)
+Fix #8775: [Win32] Don't create the main window when alt-tabbing back into fullscreen (#8792)
```
Comment (LordAro) [inline changelog.txt]: I'd suggest "Alt-Tabbing" for consistency with Alt+Enter above, but that feels a bit off. Perhaps "switching window" ?

File: changelog.txt [hunk 17]
```diff
@@ -1,3 +1,37 @@
...
+Fix #8774: Black screenshots when using 40bpp-blitter (#8791)
+Fix: [OSX] Hide dock when entering fullscreen (#8789)
+Fix: Bootstrap was only showing a black screen (#8788)
```
Comment (LordAro) [inline changelog.txt]: Tense is wrong and/or different to the others

File: changelog.txt (hunk 13 above)
Comment (TrueBrain) [reply changelog.txt:12]: Nah, it is fine. We shouldn't make this too difficult for ourselves.

File: changelog.txt (hunk 16 above)
Comment (TrueBrain) [reply changelog.txt]: Pretty sure everyone (like really: everyone) understands Alt+Tabbing :)

File: changelog.txt (hunk 17 above)
Comment (ldpl) [reply changelog.txt]: I don't think many players know what bootstrap is in OpenTTD

File: changelog.txt (hunk 10 above)
Comment (TrueBrain) [reply changelog.txt]: Went with "Setting for highest resolution of sprites to use", as that is how the setting is called ingame :) This really has been debated way too much :D :D Funny, how difficult some things are to describe in a single sentence :)

**Review by frosch123 — APPROVED** [review]

**Pull request merged**